mobile-app/lib/ui/views/learn/widgets/description/description_widget_view.dart

```diff
@@ -30,33 +30,42 @@ class DescriptionView extends StatelessWidget {
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: [
                   if (instructions.isNotEmpty)
-                    Padding(
-                      padding: const EdgeInsets.all(8.0),
-                      child: Column(
-                        crossAxisAlignment: CrossAxisAlignment.start,
-                        children: [
-                          Container(
-                            padding: const EdgeInsets.fromLTRB(16, 16, 16, 8),
-                            child: const Text(
+                    Column(
```

Comment:
Why the extra `Column` here as the same `Column` before the `if` does the same thing